.changeset/wise-boxes-develop.md

```diff
@@ -0,0 +1,42 @@
+---
+'astro': minor
+---
+
+Exports a new `getActionPath()` helper from `astro:actions`
+
+The `getActionPath()` utility accepts an action and returns a URL path so you can execute an action call as a `fetch()` operation directly. This allows you to provide details such as custom headers when you call your action.
+
+This example shows how to call a defined `like` action passing the `Authorization` header and the [`keepalive`](https://developer.mozilla.org/en-US/docs/Web/API/Request/keepalive) option:
```

Comment:
```suggestion
This example adds authorization to a `like` action by passing the `Authorization` header and the [`keepalive`](https://developer.mozilla.org/en-US/docs/Web/API/Request/keepalive) option:
```
Would that be an OK way to say this? Since this is not docs, it doesn't need to be *quite* as clinical. (But probably not as casual as the blog post.)

Reply:
I don't think it's 100% correct. You don't add authorization to the action, it assumes it's already setup and you just provide some credentials basically

Reply:
OK! If you can come up with something in plain English kind of like: "This example passes credentials to an action requiring authentication, as well as an option to keep the associated request alive even if the user navigates away from the page:" then go for it! Otherwise, what you have is fine, and non-blocking!